types/tuple.hh

```diff
@@ -102,6 +102,24 @@ std::optional<View> read_tuple_element(View& v) {
     return read_simple_bytes(v, s);
 }
 
+template <FragmentedView View>
+bytes_opt get_nth_tuple_element(View v, size_t n) {
+    for (size_t i = 0; i < n; ++i) {
+        if (v.empty()) {
+            return std::nullopt;
+        }
+        read_tuple_element(v);
```

Comment:
Potential for optimization: skip_tuple_element, to avoid constructing and then destroying a bytes_opt. Let's defer it though.

Reply:
`read_tuple_element` reads a view, not a `bytes_opt`, so it's unlikely to be an optimization.

Reply:
It reads a view but returns a byte_opt, so it has to allocate storage for the return value.

Reply:
No, `read_tuple_element` returns a view. I think you have misread it.

Reply:
Yes.